manifests/web.pp

```diff
@@ -337,6 +337,29 @@
         ],
       }
     }
+    'CentOS', 'RedHat': {
```

Comment:
why did you add RedHat here? Are you running it on RedHat enterprise?

Reply:
Per the Zabbix documentation (https://www.zabbix.com/documentation/current/manual/installation/upgrade/packages/rhel_centos), the package changes apply to both CentOS 7 and RHEL 7 and due to the surrounding case statement evaluating $facts['os']['name'], I matched the pattern present in line 306 for defining the package to be the same on Ubuntu and Debian. That was the simplest way I could think of to include RHEL without refactoring the entire case statement to use a different fact.

Leaving out RedHat would resolve the issue I was making the change for on CentOS 7, but leave the issue present if someone was utilizing RHEL 7, because the official packages for CentOS 7 and RHEL 7 are the same.